sass/_cover.scss

```diff
@@ -1,138 +1,134 @@
 .cover {
 	background-color: $background-color-cover;
-    color: #fff;
-    overflow: hidden;
-    padding: 156px 0 80px;
-    position: relative;
+	color: #fff;
+	overflow: hidden;
+	padding: 156px 0 80px;
+	position: relative;
 	transition: opacity .25s;
 	z-index: $z-index-cover;
-	
+
 	&-header {
 		@include site-width;
 		position: relative;
-		
-        p {
-            color: #fff;
-        }
-        
+
+		p {
+			color: #fff;
+		}
+
 		a {
-            color: #fff;
-            
-            &:hover {
-                border-color: #fff;
-            }
-            
-            &.author:hover {
-                border-color: transparent;
-                
-                span {
-                    border-color: #fff;
-                }
-            }
-        }
-        
-        .taxonomy-description {
+			color: #fff;
+
+			&:hover {
+				border-color: #fff;
+			}
+
+			&.author:hover {
+				border-color: transparent;
+
+				span {
+					border-color: #fff;
+				}
+			}
+		}
+
+		.taxonomy-description {
 			border-top: 1px solid rgba(255, 255, 255, .25);
-            margin-top: 20px;
-			
-            p {
+			margin-top: 20px;
+
+			p {
 				margin-bottom: 0;
 			}
 		}
-        
+
 		.search {
-			
-            &-form {
+
+			&-form {
 				margin-top: 60px;
-                @include site-width($width-narrow);
+				@include site-width($width-narrow);
 			}
-			
-            &-field {
+
+			&-field {
 				border: 0;
 				border-radius: 0;
-                box-sizing: border-box;
-                padding: 10px;
-                width: 100%;
+				box-sizing: border-box;
+				padding: 10px;
+				width: 100%;
 			}
 		}
 	}
-    
-    &-title {
-        color: #fff;
-        @include rem(font-size, 58px);
-        line-height: 86px;
-        margin: 0;
-        
-        a {
-            color: #fff;
-            font-weight: normal !important;
-        }
-
-        .page & {
-            margin-bottom: 0;
-        }
-
-        @media screen and (max-width: $width-tablet) {
-            @include rem(font-size, 40px);
-            line-height: 50px;
-        }
-    }
-    
-    &-subtitle,
+
+	&-title {
+		color: #fff;
+		@include rem(font-size, 58px);
+		line-height: 86px;
+		margin: 0;
+
+		a {
+			color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.